QUEST.tsv

```diff
@@ -2777,67 +2777,67 @@ QUEST_20151016_002784
+QUEST_20151022_002823	Detector is indicating it's far away in the upper left corner.
 QUEST_20151022_002824	
-QUEST_20151022_002825	
-QUEST_20151022_002826	
-QUEST_20151022_002827	
-QUEST_20151022_002828	
-QUEST_20151022_002829	
-QUEST_20151022_002830	
-QUEST_20151022_002831	
-QUEST_20151022_002832	
-QUEST_20151022_002833	
-QUEST_20151022_002834	
-QUEST_20151022_002835	
-QUEST_20151022_002836	
-QUEST_20151022_002837	
-QUEST_20151022_002838	
-QUEST_20151022_002839	
-QUEST_20151022_002840	
-QUEST_20151022_002841	
-QUEST_20151022_002842	
-QUEST_20151022_002843	
-QUEST_20151022_002844	
-QUEST_20151022_002845	
-QUEST_20151022_002846	
-QUEST_20151022_002847	
-QUEST_20151022_002848	
+QUEST_20151022_002825	Besides, those tombstones! It's suppose to release crimson energy.{nl}We were gathering holy energy to offer to the goddesses!{nl}
+QUEST_20151022_002826	Thank you for helping.{nl}We can take care of the rest.{nl}Let me thank you again.
+QUEST_20151022_002827	Requirement to create scripture is faith's
+QUEST_20151022_002828	They say what is the use in the world with disappeared goddesses.{nl}But we believe Goddesses will one day return and save us!{nl}
+QUEST_20151022_002829	Is it wrong to pray so that the day will come sooner?{nl}
+QUEST_20151022_002830	These tombstones were built with our devotion from our praying hearts.{nl}Until some strange person came and destroyed the tombstones, it was used for all our rites!
+QUEST_20151022_002831	It's suppose to release crimson energy, it's not a corruption!{nl}We were gathering holy energy to offer to the goddesses!{nl}
+QUEST_20151022_002832	Since you carry such malice, monsters must be attracted to you!{nl}The reason! It was created that way!{nl}
+QUEST_20151022_002833	Working under someone who neither understand or can hear!{nl}Did the person who destoryed our tombstones saw it as a corruption?
+QUEST_20151022_002834	Do you finally understand?{nl}What your utter stupidity has caused?{nl}
+QUEST_20151022_002835	If you understand, start trying to think of a way to revert this.{nl}This can't simply be ignored.
+QUEST_20151022_002836	I got a request for investigation.{nl}It's the person disguised as Anastospa and destorying Nestospa tombstones.{nl}
+QUEST_20151022_002837	The name is Alvidas. He was at Liugas Access Road.{nl}It might be dangerous, take this flare. Be careful.
+QUEST_20151022_002838	Healing potion needed for a dog.
+QUEST_20151022_002839	That's a healing potion. I will feed it to the dog when you hand it over.
+QUEST_20151022_002840	I will head to where Rose is residing.
+QUEST_20151022_002841	At first glance, it may look light, but it carries the profundity of road of swordsman.
+QUEST_20151022_002842	I hope skilled swordsman that can help the inhabitants of this area increase.
+QUEST_20151022_002843	Offense is the best defense.{nl}There's nothing to fear if you can handle two-handed sword to blow away enemies like fallen leaves of the automn wind.
```

Comment:
Offense=Attack? (or does offense also mean attack?)
if you can handle two-handed sword -> if you can handle a two-handed sword